jdaviz/core/marks.py

```diff
@@ -119,6 +119,8 @@ def set_x_unit(self, unit=None):
         self.xunit = unit
 
     def set_y_unit(self, unit=None):
+        if self.__class__.__name__ == 'SpectralLine':
```

Comment:
Is there a reason why `SpectralLine` cannot just override this method itself?

https://github.com/spacetelescope/jdaviz/blob/f552510ee17898fbac60ec3f7d7f6e7819c14912/jdaviz/core/marks.py#L204

Reply:
It is now overridden!